classes/Category.php

```diff
@@ -780,6 +787,45 @@ public static function getSimpleCategories($idLang)
 		ORDER BY c.`id_category`, category_shop.`position`');
     }
 
+    /**
+     * Get a simple list of categories with id_category, name and id_parent infos
+     * It also takes into account the root category of the current shop
+     *
+     * @param int $idLang Language ID
+     *
+     * @return array|false|mysqli_result|null|PDOStatement|resource
```

Comment:
are you sure it is all types you may return? :trollface:

Reply:
mmm perhaps mysql_result is missing here :trollface: